lib/core/lib/ops/policyOps.js

```diff
@@ -304,11 +304,16 @@ function buildPolicyOps (db, config) {
         ),
         policies_from_organization AS (
           SELECT
-            policy_id
+            op.policy_id
           FROM
-            organization_policies
+            organization_policies op
+          LEFT JOIN
+            users u ON u.org_id = op.org_id
           WHERE
-            org_id = ${organizationId}
+            u.id = ${userId}
+        ),
+        is_root_user AS (
```

Comment:
Why don't you use this directly with the `EXISTS` clause below? This query is actually returning a user entry for the user is not really `is_root_user`.
But this is not necessary ;)